node/src/main/scala/com/wavesplatform/api/http/AddressApiRoute.scala

```diff
@@ -267,8 +269,19 @@ case class AddressApiRoute(settings: RestAPISettings, wallet: Wallet, blockchain
   @Path("/data/{address}")
   @ApiOperation(value = "Complete Data", notes = "Read all data posted by an account", httpMethod = "GET")
   @ApiImplicitParams(Array(new ApiImplicitParam(name = "address", value = "Address", required = true, dataType = "string", paramType = "path")))
-  def getData: Route = (path("data" / Segment) & get) { address =>
-    complete(accountData(address))
+  def getData: Route = (path("data" / Segment) & parameter('matches.?) & get) { (address, maybeRegex) =>
```

Comment:
Please add some documentation for `matches` param (make sure to mention encoding schemes, etc.)